docs/changelog_de.md

```diff
@@ -5,6 +5,7 @@
 * Fix [#4](https://g1cp.org/issues/4): Zuvor besiegte NSCs fahren den Spieler nun nicht mehr an, wenn er ihnen im Weg steht.
 * Fix [#50](https://g1cp.org/issues/50): Die unerreichbare Truhe in der Gruft unter dem Stonehenge ist nun korrekt positioniert und zugänglich.
 * Fix [#52](https://g1cp.org/issues/52): Der Schleifstein im Neuen Lager setzt zur Benutzung nun korrekt eine Schwertklinge voraus.
+* Fix [#132](https://g1cp.org/issues/132): Die Beschreibung des Buchs "Die Gruft" lautet nun korrekt "Die Gruft" statt "Kampfkunst".
```

Comment:
I updated the changelog here to distinguish the description from the name of the item.